src/ts/spatialnavigation/TypeGuards.ts

```diff
@@ -16,10 +15,6 @@ export function isContainer(obj: unknown): obj is Container<unknown> {
   return obj !== null && obj !== undefined && obj instanceof Container;
 }
 
-export function isListBox(obj: unknown): obj is ListBox {
-  return obj instanceof ListBox;
-}
-
```

Comment:
This special handling for a `ListBox` is no longer needed as it is already covered by the `SettingsPanel` support.